libraries/AP_Scripting/docs/docs.lua

```diff
@@ -3136,6 +3136,13 @@ function arming:disarm() end
 -- It provides estimates for the vehicles attitude, and position.
 ahrs = {}
 
+-- supply an external position estimate to the AHRS (supported by EKF3)
+---@param location Location_ud -- estimated location, altitude is ignored
+---@param accuracy number -- 1-sigma accuracy in meters
+---@param timestamp_ms uint32_t_ud -- timestamp of reading in ms since boot
```

Comment:
```suggestion
---@param timestamp_ms uint32_t_ud|integer|number -- timestamp of reading in ms since boot
```
Because of the magic casting we do to uint32.

Reply:
Done